templates/results-row.php

```diff
@@ -14,5 +14,10 @@
 	<td>
 		{{data.message}}
 	</td>
+	<# if ( data.link ) { #>
+		<td>
```

Comment:
I don't think we can output the `<td>` conditionally, it either needs to be always there or never. Otherwise, we'll create invalid table markup.

I think maybe we should handle this differently: `data.link` is realistically always going to be set for every error/warning, _except_ when the current user cannot `edit_plugins`. So maybe we should check this more holistically before output the individual table as a whole.